Webex Connect Flows/v3.0/Template/Media Specific Workflows/README.md

```diff
@@ -0,0 +1,84 @@
+
+**Media Specific Workflows** : These flows need to be imported once per asset of the specific channel.
+* **Facebook Inbound Flow** - This workflow will be triggered for every inbound customer message over the integrated Facebook page.
+* **Live Chat Inbound Flow** - This workflow will be triggered for every inbound customer message over the configured customer chat widget, and the widget has a live chat form configured.
+* **Live Chat Inbound Flow Without Form** - This workflow will be triggered for every inbound customer message over the configured customer chat widget and the widget doesn't have a live chat form configured.
+* **SMS Inbound Flow** - This workflow will be triggered for every inbound customer sms sent over the integrated SMS phone number.
+* **Email Inbound Flow** - This workflow will be triggered for every inbound customer email to the integrated email account.
+* **Whatsapp Inbound Flow** - This workflow will be triggered for every inbound customer message over the integrated whatsapp business number.
+* **Live Chat Close Flow** - This workflow will be triggered when a livechat customer has ended the chat.
+
+Note : 
+- Queue selection from queue task node is mandatory for any channel flows before making flow live.
+- Flow variables must be updated in the setting section with the correct asset details.
+
+## Updating Flow Variables
+
+Flow Variables that need to be updated under Custom Variables for different channels: -
+- LiveChat :- appId,liveChatDomain
+
+  <img width="800" alt="Settings" src="../../images/Settings.png">
+
+- Facebook :- FBpageid,appid
+
+  <img width="800" alt="Facebook Settings" src="../../images/Facebook.png">
+  
+- Whatsapp :- WANumber,appid
+
+  <img width="800" alt="Whatsapp Settings" src="../../images/Whatsapp.png">
+
+- Email    :- bizemailid
+
+  <img width="800" alt="Email" src="../../images/Email.png">
+
+### Live Chat Inbound Flow Details
+- In the Live Chat Flow, we need to select the template. For creation of template refer to https://help.imiconnect.io/docs/livechat-template
+Sample template is shown in below image
+
+  <img width="800" alt="Template" src="../../images/Template.png">
+
+- In the Live Chat/In App messaging node we have selected the message type as form and we have to choose the template from the dropdown which has the list of all the livechat templates available in that tenant
+
+  <img width="800" alt="formdropdown" src="../../images/formdropdown.png">
+
+- In this sample we have filled the template as LiveChatTemplate(the sample one shown in above image)
+
+  <img width="800" alt="Prechatform" src="../../images/Prechatform.png">
+
+- In the Receive node transition action we need to set the variable customerName and customerEmail and this customerName and customerEmail will 
+be visible on the agent desktop.
+
+- We are getting the value for customerName and customerEmail from the livechat form. In the given sample livechat template the customername and customeremail
```

Comment:
We are getting the value for customerName and customerEmail from the livechat form.  -> 

The value of customerName and customerEmail will be extracted from Live Chat form response. In the sample Live Chat template, "Name" and "Email" fields are configured. Hence, `$(n2438.inappmessaging.formFields.Name)` and `$(n2438.inappmessaging.formFields.Email)` and configured as values for customerName and customerEmail variables respectively. 
Based on your form fields, the value for the above variables will change, however; the variable names i.e customerName and customerEmail must remain intact. 

For instance, if the form fields are named as UserName and UserEmail, the customerName and customerEmail variables must be set as`$(n2438.inappmessaging.formFields.UserName)` and `$(n2438.inappmessaging.formFields.UserEmail)` in the Receive node Transition Actions.